{{cookiecutter.project_slug}}/clients/web/vue3/src/composables/Users.ts

```diff
@@ -0,0 +1,111 @@
+import {
+    AccountForm,
+    EmailForgotPasswordForm,
+    LoginForm,
+    ResetPasswordForm,
+    userApi,
+  } from '@/services/users'
+  import { useMutation, useQueryClient } from '@tanstack/vue-query'
+  import { reactive, ref } from 'vue'
+  import { useRouter } from 'vue-router'
+  import { useStore } from 'vuex'
+  import { useAlert } from '@/composables/CommonAlerts'
+  
+  export function useUsers() {
```

Comment:
Try using composables for common functionality, this may not be the exact expected implementation but it works nicely. I think ideally, as is done in react, you should have separate _hooks_ for each one.